MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -18,6 +18,7 @@ internal protected enum SurfaceType
 
 		internal int width;
 		internal int height;
+        internal int ArraySize;

```

Comment:
That was quick :) I don't think much of the prefixes for static and thread-static fields, that feels quite old-school - but I guess MS get to decide...